README.md

```diff
@@ -177,6 +177,7 @@ To submit code for this Quick Start, see the AWS Quick Start [Contributor's guid
 | <a name="input_cluster_endpoint_public_access_cidrs"></a> [cluster\_endpoint\_public\_access\_cidrs](#input\_cluster\_endpoint\_public\_access\_cidrs) | List of CIDR blocks which can access the Amazon EKS public API server endpoint | `list(string)` | <pre>[<br>  "0.0.0.0/0"<br>]</pre> | no |
 | <a name="input_cluster_identity_providers"></a> [cluster\_identity\_providers](#input\_cluster\_identity\_providers) | Map of cluster identity provider configurations to enable for the cluster. Note - this is different/separate from IRSA | `any` | `{}` | no |
 | <a name="input_cluster_ip_family"></a> [cluster\_ip\_family](#input\_cluster\_ip\_family) | The IP family used to assign Kubernetes pod and service addresses. Valid values are `ipv4` (default) and `ipv6`. You can only specify an IP family when you create a cluster, changing this value will force a new cluster to be created | `string` | `"ipv4"` | no |
+| <a name="input_cluster_kms_key_additional_admin_arns"></a> [cluster\_kms\_additional\_admin\_arns](#input\_cluster\_kms\_additional\_admin\_arns) | A list of additional IAM ARNs that should have FULL access (kms:*) in the KMS key policy. | `list(string)` | `[]` | no |
```

Comment:
cluster_kms_key_additional_admin_arns missing key under inputs please run pre-commit run -a | terraform-docs

Reply:
@ZeroDeth Fix submitted here: https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/418

Reply:
Not sure what you mean by running `pre-commit run -a | terraform-docs` - Is that something I should be running locally, or commenting in the PR, to kick off a Github action, or something?